brain_observatory_analysis/dev/data_selection_tools.py

```diff
@@ -0,0 +1,139 @@
+import os
+import json
+# import warnings
+import numpy as np
+import pandas as pd
+from pathlib import Path
+
+# import logging
+# logger = logging.getLogger(__name__)
+
+
+def add_layer_column(df):
```

Comment:
There is bisect_layer from experiment_loading.
Also, this can be misleading sometimes because the imaging depth can vary for the same plane, and can criss-cross the threshold. 
We have depth_order as well, which calculates the order of depth in the same targeted_structure in a session, so it is less error-prone. Also this can be easily generalized to other types of imaging data collection (e.g., 4 depths 2 areas configuration).

Reply:
@jkim0731 I don't see it in experiment_loading.py file. Has it not been added?